assets/js/init.js

```diff
@@ -1,12 +1,15 @@
-window.Herman = (function init(Herman, $) {
-  'use strict';
-  $(() => {
-    window.hljs.initHighlighting();
-    Herman.initializeToggles();
-    Herman.initializeTabs();
-    Herman.initializeIframes();
-    Herman.getSearchData();
-  });
-
-  return Herman;
-})(window.Herman || {}, window.jQuery);
+// Add ES2015 polyfills
+import '@babel/polyfill';
```

Comment:
Are there import side effects here or something?

Reply:
Yes. Any time you see an `import` without exposing a variable name (e.g.`import 'bar'` vs `import foo from 'bar'`), you can assume there's a side effect or something exposed on the global window. In this case, Babel reads our `.browserslistrc` file to see what browsers we support, and adds the necessary polyfills. See https://github.com/babel/babel/tree/master/packages/babel-preset-env#usebuiltins-entry